content/en/docs/install-setup/installing/boot/git-provider.md

```diff
@@ -1,9 +1,11 @@
 ---
-title: Configure Git Provider
-linktitle: Configure Git Provider
-description: Configure Git providers supported by Jenkins X
+title: Configure SCM / Git
```

Comment:
Why don't you want to use Git Providers? You are not configuring Git really here.

Reply:
My personal preference would be for `Configure SCM Provider` - longstanding term-of-art for GitHub et al. But the "Provider" bit is, IMO, needed.

Reply:
> Configure SCM Provider

I could live with that. It is more generic and potentially we support another SCM at some stage (conceptually).

> But the "Provider" bit is, IMO, needed.

+1, this is my point as well.

Reply:
@ethangj - should the title not change as well?